genome_sampler/plugin_setup.py

```diff
@@ -148,7 +148,9 @@ def _3(fmt: GISAIDDNAFASTAFormat) -> DNASequencesDirectoryFormat:
         'seed': Int % Range(0, None),
     },
     outputs=[('selection', FeatureData[Selection])],
-    input_descriptions={},
+    input_descriptions={
+        'context_seqs': 'The context sequences to be sampled from.'
```

Comment:
Maybe add "Providing this will restrict the IDs sampled to only those which have an associated sequence"?